HeavyIonsAnalysis/JetAnalysis/src/HiInclusiveJetAnalyzer.cc

```diff
@@ -691,6 +700,12 @@ HiInclusiveJetAnalyzer::beginJob() {
       t->Branch("gendphijt",jets_.gendphijt,"gendphijt[ngen]/F");
       t->Branch("gendrjt",jets_.gendrjt,"gendrjt[ngen]/F");
 
+      //for reWTA reclustering
+      if(doWTARecluster_){
+        t->Branch("WTAgeneta",jets_.WTAgeneta,"WTAgeneta[nref]/F");
```

Comment:
@jusaviin @FHead should this be ngen? if yes, please also fix this in 758p3

Reply:
Yes, it should be ngen. Thank you for noticing. I will correct that here and in 758p3.